wrangling.Rmd

```diff
@@ -1260,23 +1234,21 @@ summarize(region_lang_na,
           max_most_at_home = max(most_at_home, na.rm = TRUE))
 ```
 
-### Calculating summary statistics on columns for groups of rows
+### Calculating summary statistics for groups of rows
 
-A common pairing with `summarize` is `group_by`. Pairing these functions
+A common pairing with `summarize` is `group_by`. Pairing these functions \index{group\_by}
 together can let you summarize values for subgroups within a data set,
 as illustrated in Figure \@ref(fig:summarize-groupby). 
+For example, we can use `group_by` to group the regions of the `tidy_lang` data frame
+and then calculate the minimum and maximum number of Canadians 
+reporting the language as the primary language at home 
+for each of the regions in the data set.
 
-```{r summarize-groupby, echo = FALSE, message = FALSE, warning = FALSE, fig.cap = "`summarize` + `group_by` is useful for calculating summary statistics on one, or more, column(s), for each group. It creates a new data frame with a row for each group, containing the summary statistic(s) for each column being summarized. It also creates a column listing the value of the grouping variable. The darker, top row of each table represents the column headers. The grey, blue and green coloured rows correspond to the rows that belong to each of three groups being represented in this cartoon example.", out.width="100%", fig.retina = 2}
+```{r summarize-groupby, echo = FALSE, message = FALSE, warning = FALSE, fig.cap = "`summarize` + `group_by` is useful for calculating summary statistics on one or more column(s) for each group. It creates a new data frame&mdash;with one row for each group&mdash;containing the summary statistic(s) for each column being summarized. It also creates a column listing the value of the grouping variable. The darker, top row of each table represents the column headers. The grey, blue and green coloured rows correspond to the rows that belong to each of three groups being represented in this cartoon example.", out.width="100%", fig.retina = 2}
```

Comment:
Again I think it would help to fill in values for this table. Like show the regions and stuff like that 
Also the formatting of the figure:

<img width="656" alt="Screen Shot 2021-09-29 at 5 16 32 PM" src="https://user-images.githubusercontent.com/3838091/135365568-e74a0d92-10f4-49ca-b787-bb688f2ac60d.png">

Reply:
agreed that it would be good to think about, but deferred to a later version or at least a later PR. made an issue for this already. 

formatting issues punted to style guide formatting pass.